rai_core_flask/rai_core_flask/flask_helper.py

```diff
@@ -24,13 +24,15 @@
 class FlaskHelper(object):
     """FlaskHelper is a class for common Flask utilities used in dashboards."""
 
-    def __init__(self, ip=None, port=None, with_credentials=False):
+    def __init__(self, ip=None, port=None, with_credentials=False,
+                 is_private_link_aml_workspace=False):
         # The name passed to Flask needs to be unique per instance.
         self.app = Flask(uuid.uuid4().hex)
 
         self.port = port
         self.ip = ip
         self.with_credentials = with_credentials
+        self.is_private_link_aml_workspace = is_private_link_aml_workspace
```

Comment:
to pass tests, I think you actually need to break this into several PRs:
1.) update rai-core-flask
2.) release rai-core-flask
3.) update raiwidgets